backend/app/constants/wikipedia_base_url.py

```diff
@@ -0,0 +1,2 @@
+# A constant that holds the base URL for Wikipedia articles.
+WIKIPEDIA_BASE_URL = "https://en.wikipedia.org/wiki/"
```

Comment:
I'd just have a single `constants.py`.

Reply:
I have added a single file for the constants.